api/src/wfl/api/handlers.clj

```diff
@@ -65,10 +65,12 @@
   "Return the workflows managed by the workload."
   [request]
   (log/info (select-keys request [:request-method :uri :parameters]))
-  (let [uuid (get-in request [:path-params :uuid])]
+  (let [uuid (get-in request [:path-params :uuid])
+        {:keys [status]} (get-in request [:parameters :query])]
     (jdbc/with-db-transaction [tx (postgres/wfl-db-config)]
-      (->> (workloads/load-workload-for-uuid tx uuid)
-           (workloads/workflows tx)
+      (->> (let [w (workloads/load-workload-for-uuid tx uuid)]
+             (cond status (workloads/workflows-by-status tx w status)
+                   :else  (workloads/workflows tx w)))
```

Comment:
I feel like this is similar to `if`, is there a special consideration behind using `cond`?

Reply:
(Not PR feedback but sharing as a FYI!)

I know we talked about this in stand-up a bit, but here's how I usually choose between `if`, `when`, and `cond`.

`if` - test one condition with a fallback (the "else" branch).
```
(if true "yay" "womp")
=> "yay"
(if false "yay" "womp")
=> "womp"
```

`if` - test one condition with `nil` as the fallback.
```
(when true "yay")
=> "yay"
(when false "yay")
=> nil
```

`cond` - test multiple conditions.  I introduced a `cond` with a single test case in `executor.clj` but it was because I anticipated a future in which we would be supporting other conditions.
```
(let [check "yay"]
  (cond (= check "yay") "party"
        (= check "womp") "stay home"
        :else "try again"))
=> "party"
(let [check "womp"]
  (cond (= check "yay") "party"
        (= check "womp") "stay home"
        :else "try again"))
=> "stay home"
(let [check "something"]
  (cond (= check "yay") "party"
        (= check "womp") "stay home"
        :else "try again"))
=> "try again"
```

Oftentimes when I'm using an `if` or a `when` I end up preferring `if-let` or `when-let`.  They're good ones to look at in the cheat sheet:
https://clojuredocs.org/clojure.core/if-let
https://clojuredocs.org/clojure.core/when-let

Reply:
I changed this to a an if. Thank you for the advice.